build.gradle

```diff
@@ -34,19 +34,16 @@ test {
 }
 
 dependencies {
-  implementation(platform("com.netflix.spinnaker.kork:kork-bom:$korkVersion"))
-  compileOnly "org.projectlombok:lombok"
-  annotationProcessor platform("com.netflix.spinnaker.kork:kork-bom:$korkVersion")
-  annotationProcessor "org.projectlombok:lombok"
-  testAnnotationProcessor platform("com.netflix.spinnaker.kork:kork-bom:$korkVersion")
-  testAnnotationProcessor "org.projectlombok:lombok"
+  implementation(platform("org.spockframework:spock-bom:1.3-groovy-2.5"))
+  implementation(platform("org.junit:junit-bom:5.6.2"))
 
-  implementation "com.netflix.frigga:frigga"
+  compileOnly "org.projectlombok:lombok:1.18.10"
+  annotationProcessor "org.projectlombok:lombok:1.18.10"
+  implementation "com.netflix.frigga:frigga:0.24.0"
 
-  testImplementation "org.codehaus.groovy:groovy-all"
+  testImplementation "org.codehaus.groovy:groovy-all:2.5.10"
```

Comment:
I bet you could ditch this and just transitively get whatever groovy the spock is bringing along